### PR TITLE
chore: Add 1.2.2 release notes

### PR DESCRIPTION
**Pull request by jkciesluk**

### CODE REVIEW

File: website/blog/2024-02-15-bismuth.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+## Fix sbt builds on Windows
+
+Previous release unintentionally broke some sbt builds on Windows. In this release we revert the breaking changes.
+
+## Miscellaneous
+
+- bugfix: Deduplicate references results. [kasiaMarek](https://github.com/kasiaMarek)
+- bugfix: Don't rename package when it did not change. [kasiaMarek](https://github.com/kasiaMarek)
+- bugfix: properly resolve build targets for references search
+  [\#6103](https://github.com/scalameta/metals/pull/6103)
```
Comment (kasiaMarek) [inline website/blog/2024-02-15-bismuth.md:55]: Not needed.

**Review by kasiaMarek — APPROVED** [review]
what about `mimaPreviousArtifacts`?

File: website/blog/2024-02-15-bismuth.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from contributors from the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
```
Comment (tgodzik) [inline website/blog/2024-02-15-bismuth.md:42]:
```suggestion
```

File: website/blog/2024-02-15-bismuth.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+- [Fix sbt builds on Windows](#fix-sbt-builds-on-windows)
+
+## Fix sbt builds on Windows
+
+Previous release unintentionally broke some sbt builds on Windows. In this release we revert the breaking changes.
```
Comment (tgodzik) [inline website/blog/2024-02-15-bismuth.md:48]:
```suggestion
Previous release unintentionally stopped some sbt projects on Windows from importing. In this release we revert the breaking changes.
```

**Review by tgodzik — APPROVED** [review]
Thanks! I added just two small comments

**Pull request merged**